README.md

```diff
@@ -86,7 +86,10 @@ Included in the repo is an IRC bot capable of relaying adminhelps to a specified
 
 ### LICENSE
 
-All code is licensed under the [GNU GPL v3.0](https://www.gnu.org/licenses/gpl-3.0.html) unless specified otherwise.
+All code after [commit 8cfadfbf0085c29423d2a5677db41df3ef1f59eb on 2024/11/9 at 11:42 AM EST](https://github.com/vgstation-coders/vgstation13/commit/8cfadfbf0085c29423d2a5677db41df3ef1f59eb) is licensed under [GNU AGPL v3](http://www.gnu.org/licenses/agpl-3.0.html).
```

Comment:
No need to autodefault to AGPL. You can just port AGPL code and leave home grown code as gpl since going from gpl -> agpl is possible and only 1 way.